package.json

```diff
@@ -26,44 +26,39 @@
     "release": "yarn build && auto shipit",
     "start": "concurrently \"yarn build:watch\" \"yarn storybook -- --no-manager-cache --quiet\"",
     "storybook": "start-storybook -p 6006",
-    "test": "jest"
+    "test": "vitest run",
+    "test:watch": "vitest dev"
   },
   "lint-staged": {
     "*.{ts,js,css,md}": "prettier --write"
   },
-  "devDependencies": {
-    "@babel/cli": "^7.12.1",
+  "dependencies": {
     "@babel/core": "^7.12.3",
     "@babel/parser": "^7.12.11",
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-typescript": "^7.13.0",
     "@babel/traverse": "^7.12.11",
     "@babel/types": "^7.14.8",
```

Comment:
If we are doing a major release, let's also update all babel dependencies

Reply:
Sure thing! They only upgraded in minor though